specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/namespaces-preview.json

```diff
@@ -0,0 +1,1339 @@
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PrivateLinkResourcesListResult": {
+      "description": "Result of the List private link resources operation.",
+      "properties": {
+        "value": {
+          "description": "A collection of private link resources",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        },
+        "nextLink": {
+          "description": "A link for the next page of private link resources.",
+          "type": "string"
+        }
+      }
+    },
+    "userAssignedIdentityProperties": {
+      "x-ms-client-flatten": true,
+      "type": "object",
+      "properties": {
+        "userAssignedIdentity": {
+          "type": "string",
+          "description": "ARM ID of user Identity selected for encryption"
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationList": {
+      "description": "Result of the List NetworkSecurityPerimeterConfiguration operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of NetworkSecurityPerimeterConfigurations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          }
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfiguration": {
+      "description": "Network Security Perimeter related configurations of a given namespace",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkSecurityPerimeterConfigurationProperties",
+          "description": "Properties of the Network Security Perimeter Configuration",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "description": "Fully qualified identifier of the resource",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the resource",
+          "type": "string"
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationProperties": {
+      "description": "Properties of NetworkSecurityPerimeterConfiguration",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of NetworkSecurityPerimeter configuration propagation",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Updating",
+            "Accepted",
+            "InvalidResponse",
+            "Succeeded",
+            "SucceededWithIssues",
+            "Failed",
+            "Deleting",
+            "Deleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "NetworkSecurityPerimeterConfigurationProvisiningState",
+            "modelAsString": true
+          }
+        },
+        "provisioningIssues": {
```

Comment:
>provisioningIssues [](http://example.com/codeflow?start=9&length=18)

Do you intend this to be the list of provisioningIssues encountered with the latest attempt?

Reply:
Does it make sense to return a list of issues that happened in the past with every Get call for this resource?

Reply:
Do you need the user to specify this or should this be readonly?

Reply:
This whole payload NetworkSecurityPerimeterConfigurations is readonly as it is used only in one GET. I will update

Reply:
Marked the payloads of GET reponses are readonly.

Reply:
Provisioning Issues represents list of issues on last workflow attempt to update the resource. It doesn't include past issues for now. Please check the email thread for more detail.